src/p5.dance.js

```diff
@@ -1046,14 +1048,31 @@ module.exports = class DanceParty {
       this.p5_.pop();
     }
 
-    this.p5_.fill("black");
-    this.p5_.textStyle(this.p5_.BOLD);
-    this.p5_.textAlign(this.p5_.TOP, this.p5_.LEFT);
-    this.p5_.textSize(20);
-
     this.world.validationCallback(this.world, this, this.sprites_, events);
     if (this.showMeasureLabel && this.getCurrentMeasure() >= 1) {
-      this.p5_.text(`${this.i18n.measure()} ${Math.floor(Math.max(0, this.getCurrentMeasure()))}`, 10, 20);
+      const text = `${this.i18n.measure()} ${Math.floor(Math.max(0, this.getCurrentMeasure()))}`;
```

Comment:
@Hamms Should localising this text have been done with token substitution so that it works in all locales?

Reply:
arguably yes, but for something as simple as appending a number to a string I believe this should work the same in all locales